### ISSUE TITLE
Add Glossary feedback link to SPARC Portal footer

### ISSUE THREAD

**Issue by KZzizzle** [ITISFoundation/osparc-issues#201]
##### Acceptance Criteria:

* Add Glossary Feedback Link to Glossary documentation article

##### Please leave your contact information
**Name**
Sue Tappan

**Email**
susan@mbfbioscience.com

**Can you provide a short description of what you were doing?**
Checking out the Glossary page 

**Can you provide specific and detailed description?**
The Site Feedback link at the footer links to the generic portal feedback form, but there is a specific feedback form for glossary information. It should link to that. 

**How would you like the experience to improve?**
Change the Site Feedback link to https://www.wrike.com/form/eyJhY2NvdW50SWQiOjMyMDM1ODgsInRhc2tGb3JtSWQiOjI3MTY2N30JNDczODIyNTcxODA4OQlmMzMxOTUxNTk2YTA2MmJhYjExNWE3MjdkZjY4MjYwZGQ4NTBkZGE3MmU5NzYyMzVkM2ZjNmE0NzIxOGM2NTk4

**Click the box if you would like us to keep you in the loop as we work to resolve this issue.**
##### Tell us where you found the issue and what you were trying to do.
**My feedback is about:**
SPARC Documentation page

**Comment by KZzizzle** [comment]
➤ DRC SIMCORE commented:

MBF SPARC, I think this would not be straightforward since I presume the same footer is added to every page and hence the same wrike link. Leonardo Guercio and Jeremy Lecoeur can confirm this. Since it may not be easy (nor intuitive) that the same footer link goes to different places depending on the rest of the page, perhaps one solution is to have another link in the footer that specifically is labeled as "glossary feedback"?

**Comment by KZzizzle** [comment]
➤ MBF SPARC commented:

DRC SIMCORE works for me.  i think location is important, since people who may submit feedback for the glossary may do so because they also know and   have submitted feedback on the main page.

**Comment by KZzizzle** [comment]
➤ Leonardo Guercio commented:

MBF SPARCDRC SIMCORE Confirm that the feedback link in the footer is fixed on every page. I like the solution of having a separate glossary feedback link. I think it would be helpful to have Gazelle weigh in on the best location for this since. Ellen M. Goldstein - CX Any thoughts?

**Comment by KZzizzle** [comment]
➤ Ellen M. Goldstein - CX commented:

Leonardo Guercio Glossary Feedback can go right under the Site Feedback link in the footer for each page, for maximum flexibility, as people may want to add an item from any page.  MBF SPARC  - The page is written elegantly!

**Comment by KZzizzle** [comment]
➤ MBF SPARC commented:

Ellen M. Goldstein - CX i can't take credit for that, Sonia Sharma or Maryann Martone get the high fives from you and me!

**Comment by KZzizzle** [comment]
➤ DRC SIMCORE commented:

Jeremy Lecoeur could you implement the addition of Glossary feedback to the footer per Ellen's suggestion?

**Comment by KZzizzle** [comment]
➤ Cameron Baney commented:

Leonardo Guercio 1 point

**Comment by KZzizzle** [comment]
➤ Charles Horn commented:

I'm not sure why this would be needed. Can't this be captured by Site Feedback? This seems too specific specific for the footer.

**Comment by KZzizzle** [comment]
➤ Maryann Martone commented:

Disagree with Charles Horn SPARC uses a lot of jargon that is explained in the Glossary.  The Glossary is not easy to find unless you know it is there already.  Why should there be any objection to including it in the footer.

**Comment by KZzizzle** [comment]
➤ Abby Paulson commented:

CCB Review: 
Andre: The initial idea was to capture terms people were not familiar with. 
Andrew: Sue's ticket is requesting a feedback link. 
Abby: Could this go on the Glossary help page?
Consensus, agree. 

This has to come back to the Tech Leads. 

It will be added to the Glossary documentation.

**Comment by KZzizzle** [comment]
➤ Sonia Sharma commented:

David NickersonMBF SPARC (Sue) Please have a look at - https://staging.sparc.science/help/3vcLloyvrvmnK3Nopddrka Is this as expected?

**Comment by KZzizzle** [comment]
➤ David Nickerson commented:

Thanks Sonia Sharma, looks fine to me.

**Comment by KZzizzle** [comment]
➤ Sonia Sharma commented:

Thanks David Nickerson

**Comment by KZzizzle** [comment]
➤ Maryann Martone commented:

I'm a little confused about this.  There's the glossary feedback request and then there's putting a link to the glossary in the footer.  This ticket seems to conflate the two.

**Comment by KZzizzle** [comment]
➤ MBF SPARC commented:

Maryann Martone i tested the acceptance criteria. i think my original suggested improvement was determined not to be the right way forward (i.e., not put a link to the glossary feedback on every page in the portal site), but instead to make it more context sensitive to the glossary page itself.

**Comment by KZzizzle** [comment]
➤ Maryann Martone commented:

But I thought that it would be useful to have the glossary more visible on the site so that those who are browsing our materials can access it.

**Comment by KZzizzle** [comment]
➤ David Nickerson commented:

yep - I think the way this ticket evolved ended up conflating the addition of a link to the glossary from the page footer and the addition of the new feature to get feedback on the glossary. Here we have resolved the getting feedback issue here but not the easy access to the glossary. I've just added https://www.wrike.com/open.htm?id=497719970 to track the addition of the glossary link in the footer.
